wit/types.wit

```diff
@@ -110,10 +110,20 @@ interface types {
         /// length in bytes of the pathname contained in the symbolic link.
         size: filesize,
         /// Last data access timestamp.
+        ///
+        /// A value of 0 indicates the platform doesn't maintain an access
+        /// timestamp for this file.
         data-access-timestamp: datetime,
         /// Last data modification timestamp.
+        ///
+        /// A value of 0 indicates the platform doesn't maintain a
+        /// modification timestamp for this file.
+        data-modification-timestamp: datetime,
+        /// Last file status-change timestamp.
+        ///
+        /// A value of 0 indicates the platform doesn't maintain a
+        /// status-change timestamp for this file.
         data-modification-timestamp: datetime,
```

Comment:
looks like a bug?

```suggestion
        status-change-timestamp: datetime,
```

Reply:
Ah, this was a bug, but the bug was that I has copy-and-pasted an extra line. I've now fixed it.